server/src/main/java/org/opensearch/index/SegmentReplicationStats.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.io.stream.Writeable;
+import org.opensearch.core.xcontent.ToXContentFragment;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.index.shard.ShardId;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Segment Replication Stats.
+ *
+ * @opensearch.internal
+ */
+public class SegmentReplicationStats implements Writeable, ToXContentFragment {
+    @Override
+    public String toString() {
```

Comment:
Can be defined in the end of the class or after where varaible `shardStats` is defined.